.travis.yml

```diff
@@ -121,16 +121,18 @@ matrix:
     dist: trusty
     env: BUILD_TYPE=Release
     name: clang release
-  - os: osx
-    env: BUILD_TYPE=Debug
-    osx_image: xcode9.2
-    compiler: clang
-    name: osx debug
-  - os: osx
-    osx_image: xcode9.2
-    env: BUILD_TYPE=Release
-    compiler: clang
-    name: osx release
+  # macOS targets are disabled until we figure out a way to not get timeouts
+  # and are able to install Boost.python efficiently on macOS.
+  # - os: osx
+  #   env: BUILD_TYPE=Debug
```

Comment:
Is it just that installing Boost.python on macOS takes too long, or that Boost.python doesn't work?

Reply:
It'll take too long. We have to stop testing on macOS because of the timeouts. I've found a way to get around our timeouts on Linux builds but the same approach hasn't worked for me on the macOS builds. I'm cleaning up some code and will have a PR for that in later. I believe if we add a build of clang with libc++ we should be able to remain macOS compatible (include what you use also helps with that). As far as I'm aware most of the issues with SpEC running on macOS has been missing includes, which are actually a libc++ vs. libstdc++ issue not a macOS vs. Linux issue.

Reply:
> As far as I'm aware most of the issues with SpEC running on macOS has been missing includes, which are actually a libc++ vs. libstdc++ issue not a macOS vs. Linux issue.

Yep, can confirm this has been the dominant issue. Other recurring issues have been bugs in macOS libraries that need to be worked around, or different behavior of shell scripts (used in testing and/or evolution scripts) from BSD vs GNU. In other words, not really C++ issues.